src/ngraph/op/non_zero.hpp

```diff
@@ -0,0 +1,49 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v0
```

Comment:
This is a v2 op, I believe it should be in the `ngraph::op::v2` namespace to keep things tidy.

Reply:
@diyessi What do you think?

Reply:
Doesn't matter to me as long as the next version is a bigger number.

Reply:
I think we should stick to giving ops names indicating the opset they were introduced with. This will make it easier to recreate their history just by looking at the namespaces in the op definition.